.github/workflows/dotnet-runtime-80.yaml

```diff
@@ -0,0 +1,52 @@
+name: dotnet-runtime-80
+
+on:
+  push:
+    branches: main
+    paths:
+      - dotnet-runtime-80/**
+  pull_request:
```

Comment:
question(blocking): Wouldn't this publish the snap as soon as someone opens a PR? If this is true, this sounds like something that could be exploited, even though it just publishes to edge.

Reply:
Oh, overlooked that line. Good to merge, Thanks!